spec/jobs/education_form/forms/va0994_spec.rb

```diff
@@ -9,9 +9,5 @@
 
   %w[kitchen_sink prefill simple].each do |form|
     test_spool_file('0994', form)
-  end
-
-  %w[kitchen_sink_old].each do |form|
-    test_spool_file('0994', form, [:edu_form_omb_and_expiration])
-  end
+  ends
```

Comment:
typo here

Reply:
Good catch, fixed this in the latest commit.